components/form/checkboxesWithParent/README.md

```diff
@@ -0,0 +1,23 @@
+# FormCheckboxesWithParent
+
+> Description
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/form-checkboxes-with-parent --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import FormCheckboxesWithParent from '@schibstedspain/form-checkboxes-with-parent'
+
+return (<FormCheckboxesWithParent />)
+```
+
```

Comment:
If the component has styling is a good idea to add somewhere that you must to import styles to in `index.scss` file:
```css
// Add component styles to index.scss file:
@import '~@schibstedspain/form-checkboxes-with-parent/lib/index'
```

Reply:
README updated with more information :)